.github/workflows/build-nginx.yml

```diff
@@ -0,0 +1,19 @@
+name: Build Containers
+
+on:
+  push:
+    branches: ["**"]
```

Comment:
if we keep following the "fast-forward to main" process for our release, we may want to exclude the `main` branch here